index.yaml

```diff
@@ -1356,6 +1356,35 @@ entries:
     urls:
     - assets/rancher-external-ip-webhook/rancher-external-ip-webhook-0.1.400.tgz
     version: 0.1.400
+  rancher-externalip-webhook:
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: External IP Webhook
+      catalog.cattle.io/namespace: cattle-externalip-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/release-name: rancher-externalip-webhook
+      catalog.cattle.io/ui-component: rancher-externalip-webhook
+    apiVersion: v1
+    appVersion: v1.0.0
+    created: "2021-07-15T10:19:03.648261425-07:00"
+    description: |
+      Deploy the externalip-webhook to mitigate k8s CVE-2020-8554
+    digest: 51418999ae77dfa5c18bbcacbf324cc5dd24a866fab6a2dc677a2d612dd62cbc
+    home: https://github.com/rancher/externalip-webhook
+    keywords:
+    - cve
+    - externalip
+    - webhook
+    - security
+    maintainers:
+    - email: raul@rancher.com
+      name: rawmind0
+    name: rancher-externalip-webhook
+    sources:
+    - https://github.com/rancher/externalip-webhook
+    urls:
+    - assets/rancher-external-ip-webhook/rancher-externalip-webhook-1.0.0.tgz
+    version: 1.0.0
```

Comment:
This is the version the scripts generate, it didn't add +up1.0.0

Reply:
right, this is expected because the value of `version` is the same as the upstream version.